packages/transforms/mocking/yaml-config.graphql

```diff
@@ -0,0 +1,44 @@
+extend type Transform {
+    """
+    Mocking configuration for your source
+    """
+    mocking: MockingConfig
+}
+
+type MockingConfig {
+    """
+    If this expression is truthy, mocking would be enabled
+    """
+    if: Boolean
+    """
+    Do not mock any other resolvers other than defined in `mocks`
+    """
+    preserveResolvers: Boolean
+    """
+    Mock configurations
+    """
+    mocks: [MockingFieldConfig]
+}
+
+type MockingFieldConfig {
+    """
+    Resolver path
+    Example: User.firstName 
```

Comment:
Add a not about wildcard if it's available

Reply:
I think wildcard shouldn't work there because all fields will have different types 90%, no?
`name, age, birthDate` etc...